stable/nginx-ingress/README.md

```diff
@@ -64,6 +64,7 @@ Parameter | Description | Default
 `controller.service.externalIPs` | controller service external IP addresses | `[]`
 `controller.service.loadBalancerIP` | IP address to assign to load balancer (if supported) | `""`
 `controller.service.loadBalancerSourceRanges` | list of IP CIDRs allowed access to load balancer (if supported) | `[]`
+`controller.service.loadBalancerTerminateSSL` | terminates SSL at load balancer (forwards 443 to 80) | false
```

Comment:
I'm not sure how I feel about the naming here. I feel like it implies that by default SSL passthrough is enabled which I don't believe is the case. Please confirm. It is my believe that all SSL sessions will terminate on the ingress controller even if the backend service is SSL enabled.